csi/shell/entrypoint.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 set -xe
 
+# Check if required subfolders exist in $KUBELET_ROOTDIR folder
+if [ ! -d "$KUBELET_ROOTDIR/pods" ]; then
+    echo "Error: $KUBELET_ROOTDIR does not contain /pods folder, it is not a kubelet rootdir."
```

Comment:
I think we can check if the KUBELET_ROOTDIR folder should contain the following files and subfolders required by kubelet:

/var/lib/kubelet/config.yaml: kubelet configuration file.
/var/lib/kubelet/pods: directory where kubelet manages pods.
/var/lib/kubelet/plugins: directory where kubelet uses plugins.
/var/lib/kubelet/plugins/kubernetes.io: directory where kubelet uses Kubernetes plugins.

Reply:
/var/lib/kubelet/plugins must exist: https://github.com/fluid-cloudnative/fluid/blob/dc882689dbbfb0bdcc3de821dfde3858ebe7036b/charts/fluid/fluid/templates/csi/daemonset.yaml#L133-L136